projects/.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
-.vscode/
\ No newline at end of file
+.vscode/
+.idea/
```

Comment:
Did you do this after you already pushed the .vscode and .idea directories? Because if not, it's weird that they are still in here.

Reply:
Noticed that the .gitignore was nested, and i opened the project on the highest level, so it didn't register :( Will do next time!